.travis.yml

```diff
@@ -7,6 +11,7 @@ services:
 script:
   - bundle exec rspec --format d
   - bundle exec codeclimate-test-reporter
+  - docker build --rm=false . 
```

Comment:
So when I ran this on CircleCI, the default `--rm=true` seemed to play wrong with their version of btrfs which generated a ton of warnings. While those warnings didn't make any difference, I was afraid that if a genuine problem occurred, they would be obscured by the cruft. Perhaps they'd even consider those warnings to be the source of the problem, and spend precious time dealing with this red herring. Let me see if you can remove it for Travis.